### PR TITLE
Adapt dekorate

### PR DESCRIPTION
**Pull request by jacobdotcosta**

### CODE REVIEW

**Review by Sgitario — CHANGES_REQUESTED** [review]
Same comments as in other reviews.

File: README.md [hunk 1]
```diff
@@ -67,6 +67,13 @@ helm uninstall healthcheck
 ./run_tests_with_dekorate.sh
 ```
 
+Alternativelly, tests can be executed against a specific Spring Boot version by passing the
+version as a `-v` parameter.
+
+```bash
+./run_tests_with_dekorate.sh -v "2.6.7"
```
Comment (Sgitario) [inline README.md]: The script needs to be updated to follow the same convention as done in the rest-http example, right?

**Review by Sgitario — CHANGES_REQUESTED** [review]
Also, missing <version>${spring-boot.version}</version>`  in the spring maven plugin.

File: README.md (hunk 1 above)
Comment (jacobdotcosta) [reply README.md]: done.

**Review by Sgitario — APPROVED** [review]
lgtm now, conflicts need to be resolved before merging

**Pull request merged**